eml/services/utilityFunctions.js

```diff
@@ -13,6 +13,18 @@ export function getDifficultyLabel(lvl) {
   }
 };
 
+
+export const convertMsToTime = (ms) => {
+  let seconds = Math.floor((ms / 1000) % 60);
+  let minutes = Math.floor((ms / (1000 * 60)) % 60);
```

Comment:
Why use `% 60` when calculating minutes? 
Correct me if im wrong, but in a edge case, inputting 60 mins would result in `00:00` and not `60:00`

```suggestion
  let minutes = Math.floor((ms / (1000 * 60));
```

Reply:
Done as well. Check newest commit